src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,4 @@
+.catalog {
+  display: flex;
+  margin-top: 50px;
```

Comment:
You could't set margin to block, only for elements. Try to use mix, by adding class page to body and `page__catalog` class to `catalog` block. And move margin there.